models/sudoswap/ethereum/sudoswap_ethereum_events.sql

```diff
@@ -9,32 +9,35 @@
 }}
 
 --base table CTEs
-WITH
+WITH 
```

Comment:
nitpick: if you turn on whitespace removal on save in your editor these will go away and the diff will be easier to review

Reply:
done :heavy_check_mark: